scripts/github-repos/README.md

```diff
@@ -1,53 +1,138 @@
+Create 10.5 repos for each child team. Repos' names are form like "PREFIX-FILENAME-[Team number]"
+Make sure child teams are formed before running this command.
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -t, --template=TEMPLATE          The repo name within the org to use as template
+    -f, --filename=FILENAME          The base filename for repos.
+    -s, --studentteam=STUDENTTEAM    The team name of all the students team
+    -g, --gsiteam=GSITEAM            The team name of all the staff team
+
+'remove_indiv_repos'
+Delete all repos whose names are formed like "PREFIX-[username]-FILENAME".
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -f, --filename=FILENAME          The base filename for repos.
+
+'remove_team_repos'
+Delete all repos whose names are formed like "PREFIX-FILENAME-[Team number]".
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -f, --filename=FILENAME          The base filename for repos.
+
+'remove_teams'
+Remove all students and child teams in STUDENTTEAM from the org.
+Remove STUDENTTEAM as well.
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -s, --studentteam=STUDENTTEAM    The team name of all the students team
+
+'remove_team_repo_access'
+Remove students access to CHIP 10.5 repos that are formed like "PREFIX-FILENAME-[Team number]".
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -f, --filename=FILENAME          The base filename for repos.
 ```
 
 This script creates a team that include all stundents in the same semester, eg fa23. 
 Then creates different child teams for them for the chip10.5. At a minimum, 
-you need a CSV file listing all enrolled students with columns 'Email' and 'Team'.  
+you need a CSV file listing all enrolled students with columns 'Username' and 'Team'.  
 The values in the Team columns should be nonnegative integers identifying teams.
 
 Org setting: Assume Base permissions is no permission, each students can only access their team repos.
 
 **Assumption:** You have a GITHUB ORG API key.
 
-### Create a team under the org that includes all students in the same semester
+### Create a team under the org that has all students in the same semester and send invitation
 
 **Use case:** Create a team called STUDENTTEAM, and send invitations 
-to the students in csv file. If STUDENTTEAM exists, delete the team and 
-create a new one. (Team repos still exist) 
-For all students NOT in that team, add/invite into that team.
+to the students in csv file. If STUDENTTEAM exists, delete STUDENTTEAM and 

```

Comment:
this contradicts the comments near the beginning of the README about what happens if the team exists.  which is correct??

Reply:
I will delete this. I believe if the STUDENTTEAM exists, sending invitation again is more reasonable.